frontend/src/metabase-lib/v1/metadata/Table.ts

```diff
@@ -87,7 +87,7 @@ class Table {
   }
 
   dateFields() {
```

Comment:
Should this be renamed to `temporalFields` and a `dateFields` and `dateOrDateTimeFields` be added?